javascript/javascript1/week4/ticTacToe.js

```diff
@@ -0,0 +1,54 @@
+// const position = [
+//   ['o', ' ', ' '],
+//   ['o', ' ', ' '],
+//   ['o', ' ', ' ']
+// ];
+// const position = [
+//   [' ', ' ', 'o'],
+//   [' ', ' ', 'o'],
+//   [' ', ' ', 'o']
+// ];
+// const position = [
+//   ['o', 'o', 'o'],
+//   [' ', ' ', ' '],
+//   [' ', ' ', ' ']
+// ];
+// const position = [
+//   [' ', ' ', ' '],
+//   ['o', 'o', 'o'],
+//   [' ', ' ', ' ']
+// ];
+// const position = [
+//   [' ', ' ', ' '],
+//   [' ', ' ', ' '],
+//   ['o', 'o', 'o']
+// ];
+// const position = [
+//   ['o', ' ', ' '],
+//   [' ', 'o', ' '],
+//   [' ', ' ', 'o']
+// ];
+// const position = [
+//   [' ', ' ', 'o'],
+//   [' ', 'o', ' '],
+//   ['o', ' ', ' ']
+// ];
+const position = [
+    [' ', 'x', ' '],
+    [' ', 'x', ' '],
+    [' ', 'x', ' ']
+  ];
+  // console.log(position[0][1])
+  // console.log(position[1][1])
+  // console.log(position[2][1])
+  function hasWon(position, player){
+      if( position[0][1] === player && position[1][1] === player  && position[2][1] === player ){
```

Comment:
something like this, except you didn't finish. There are 3 verticals, 3 horizontals and 2 diagonals.